frontend/catalyst/passes/split_non_commuting.py

```diff
@@ -0,0 +1,605 @@
+# State Evolution Outlining Implementation
+from dataclasses import dataclass, field
+from itertools import chain
+from typing import Type, TypeVar
+
+import logging
+import numpy as np
+import pennylane as qml
+
+from catalyst.passes.xdsl_plugin import getXDSLPluginAbsolutePath
+
+from pennylane.compiler.python_compiler import compiler_transform
+from pennylane.compiler.python_compiler.dialects import quantum
+
+from xdsl.context import Context
+from xdsl.dialects import builtin, func
+from xdsl.ir import Operation, SSAValue
+from xdsl.passes import ModulePass
+from xdsl.pattern_rewriter import PatternRewriter, RewritePattern
+from xdsl.rewriter import InsertPoint
+
+T = TypeVar("T")
+
+logger = logging.getLogger(__name__)
+logger.disabled = True
+
+if not logger.handlers:
+    handler = logging.StreamHandler()
+    handler.setLevel(logging.DEBUG)
+    formatter = logging.Formatter("%(asctime)s - %(name)s - %(levelname)s - %(message)s")
+    handler.setFormatter(formatter)
+    logger.addHandler(handler)
```

Comment:
You should be able to use the PennyLane logging functionality that provides default config already.

I'd change the above to use something similar to https://github.com/PennyLaneAI/catalyst/blob/9376631b57b62d0153950142b8087baa807b6b9d/frontend/catalyst/device/qjit_device.py#L51 and https://github.com/PennyLaneAI/catalyst/blob/9376631b57b62d0153950142b8087baa807b6b9d/frontend/catalyst/device/qjit_device.py#L57 with usage matching https://github.com/PennyLaneAI/catalyst/blob/9376631b57b62d0153950142b8087baa807b6b9d/frontend/catalyst/device/qjit_device.py#L145